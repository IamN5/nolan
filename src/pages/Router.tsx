import React from "react";
import { BrowserRouter } from "react-router-dom";
import Home from "./Home/Home";
import "./main.css";

function Router() {
    return (
        <>
            <BrowserRouter>
                <Home />
            </BrowserRouter>
        </>
    );
}

export default Router;
