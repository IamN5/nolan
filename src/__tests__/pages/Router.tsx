import React from "react";
import Router from "../../pages/Router";
import renderer from "react-test-renderer";

it("renders Router component", () => {
    const tree = renderer.create(<Router />).toJSON();
    expect(tree).toMatchSnapshot();
});
