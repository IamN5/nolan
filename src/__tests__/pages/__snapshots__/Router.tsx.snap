// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Router component 1`] = `
Array [
  <div
    className="ant-row navbar"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-sm-3 ant-col-md-4 ant-col-lg-4 ant-col-xl-6"
      style={Object {}}
    >
      <h1
        className="sc-htpNat gYQSzf"
      >
        Nolan
      </h1>
    </div>
    <div
      className="ant-col search-col ant-col-sm-12 ant-col-md-11 ant-col-lg-12 ant-col-xl-12"
      style={Object {}}
    >
      <form
        className="ant-form ant-form-horizontal search"
        onSubmit={[Function]}
      >
        <div
          className="ant-row ant-form-item search-form-item"
          style={Object {}}
        >
          <div
            className="ant-col ant-form-item-control-wrapper"
            style={Object {}}
          >
            <div
              className="ant-form-item-control"
            >
              <span
                className="ant-form-item-children"
              >
                <input
                  className="ant-input ant-input-lg"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Search for movies and TV shows"
                  style={
                    Object {
                      "textAlign": "center",
                    }
                  }
                  type="text"
                  value=""
                />
              </span>
            </div>
          </div>
        </div>
      </form>
    </div>
    <div
      className="ant-col ant-col-sm-9 ant-col-md-8 ant-col-lg-8 ant-col-xl-6"
      style={Object {}}
    >
      <div
        className="sc-bZQynM czlNrT"
      >
        <a
          className="sc-EHOje jJBdpb"
        >
          Login
        </a>
        <a
          className="sc-EHOje bhZWDX"
        >
          Create New Account
        </a>
      </div>
    </div>
  </div>,
  <div
    className="sc-bdVaJa bDhinl"
    width={1024}
  >
    <div
      className="sc-bwzfXH phNPS"
    >
      New Releases
    </div>
  </div>,
]
`;
